manager/orchestrator/taskreaper/task_reaper.go

```diff
@@ -272,10 +280,6 @@ func (tr *TaskReaper) tick() {
 					break
 				}
 			}
-
-			if runningTasks <= 1 {
```

Comment:
We could remove `runningTasks` completely, if we're not going to use it for anything.